certified-connectors/inQuba Journey/settings.json

```diff
@@ -0,0 +1,9 @@
+{

```

Comment:
@prinayinq Please remove the settings.json and icon.png files from your PR which are not required to be open-sourced.

Reply:
Settings.json and icon.png removed.